docs/source/tutorials/outliers.ipynb

```diff
@@ -355,62 +389,56 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fb2b6afa",
+   "id": "089d5860",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Visualize 15 least likely outliers in train_data\n",
-    "bottom_train_outlier_idxs = (-train_outlier_scores).argsort()[:15]\n",
-    "bottom_train_outlier_subset = torch.utils.data.Subset(train_data, bottom_train_outlier_idxs)\n",
-    "plot_images(bottom_train_outlier_subset)"
+    "bottom_train_ood_features_idxs = (-train_ood_features_scores).argsort()[:15]\n",
+    "bottom_train_ood_features_subset = torch.utils.data.Subset(train_data, bottom_train_ood_features_idxs)\n",
+    "plot_images(bottom_train_ood_features_subset)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "61020539",
+   "id": "2521aefb",
    "metadata": {},
    "source": [
     "#### Scoring outliers in additional test data\n",
     "\n",
-    "Now suppose we want to find outlier images in our additional test data, in particular images unlikely to stem from the same distribution as the training data. We can again use `get_outlier_scores()` this time passing in the previously-fitted estimator `knn` used to measure similarity between training examples."
+    "Now suppose we want to find outlier images in some never before seen test data, in particular images unlikely to stem from the same distribution as the training data. We can again use our fitted `OutOfDistribution` object to measure similarity between these new testing examples."
```

Comment:
Instead of "OutOfDistribution object", what if we call it "OutOfDistribution estimator"?

Reply:
"to measure similarity between these new testing examples"
->
"to score how typical each new test example would be under the the training data distribution"